bootcamp/tutorials/integration/function_calling.ipynb

```diff
@@ -0,0 +1,430 @@
+{
```

Comment:
<p>It'd be helpful to say few words explaining the idea of using pymilvus[model] lib to run embedding models locally and generate vectors.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>